go.vendor

```diff
@@ -0,0 +1 @@
+sigs.k8s.io/cluster-api
```

Comment:
What's the significance of this file?

Reply:
It’s to fully vendor cluster-api repository, see https://github.com/kubernetes-sigs/cluster-api/pull/1134